### DISCUSSION TITLE
vercel服务端部署正常，MongoDB数据库，注册失败

### DISCUSSION THREAD

**Discussion by steaminggoat** [walinejs/waline#863] [Q&A] [answered]
vercel服务端部署正常，MongoDB数据库，修改环境变量后也重新部署了，但仍然注册失败
按照要求部署，环境变量添加了邮箱自动回复，能帮忙看下原因吗？
https://hugo-comments.vercel.app

**Comment by lizheming** [comment, marked as answer]
MongoDB 是用的官网的嘛？看起来像是因为配置有点问题链接数据库超时导致的问题。

**Reply by steaminggoat** [reply]
是用得官网的，能请教下如何修复吗

**Reply by steaminggoat** [reply]
发现问题了,MongoDB我原本创建了一个专用账户给Waline使用，不知道为什么好像一直有错。换成管理员账密之后能够注册并登录了。不过不清楚能否通过专用账户进行Waline的使用，这个之后再试一试把。
